charts/featureform/charts/dashboard-metadata/values.yaml

```diff
@@ -15,6 +15,9 @@ imagePullSecrets: []
 nameOverride: ""
 fullnameOverride: ""
 
+#current application version
+featureformVersion: "v0.8.1"
```

Comment:
thoughts on using the `values.yaml` file as the version prop home? pretty standard but we can store it elsewhere too.

Reply:
updating from our side chat: 

we'll use the `.Values.global.version | default .Chart.AppVersion` approach. that way we can just set the global as needed during upgrades etc.

Reply:
and done.